python-package/lightgbm/VERSION.txt

```diff
@@ -0,0 +1 @@
+2.0.10
```

Comment:
What for do we need the duplicate of this?
https://github.com/Microsoft/LightGBM/blob/master/VERSION.txt

Reply:
I add it to git accidentally